packages/application-config/src/substitute-variable-placeholders.ts

```diff
@@ -31,6 +40,11 @@ const isIntlVariablePlaceholder = (valueOfPlaceholder: string) =>
 const isFilePathVariablePlaceholder = (valueOfPlaceholder: string) =>
   Boolean(valueOfPlaceholder.match(filePathRefSyntax));
 
+const isStructuredJson = (
```

Comment:
Could we maybe use the new exported function from the `i18n` package instead?
(_maybe we don't want to add that dependency to this package_)

Reply:
Yeah I didn't want to include an extra dependency. I think it's fine to have it duplicated for now.